pkg/kube/kubehttp.go

```diff
@@ -213,9 +213,12 @@ func (k *KubeHTTP) Apply(ctx context.Context, manifest []byte, namespace string)
 		return fmt.Errorf("failed to dynamic resource interface: %w", err)
 	}
 
+	force := true
 	_, err = dr.Patch(ctx, name, types.ApplyPatchType, data, metav1.PatchOptions{
 		FieldManager: "wego",
```

Comment:
@bigkevmcd I think we could actually avoid setting this `FieldManager` right? that might avoid the conflict too.